lib/connectors_async/job_runner.rb

```diff
@@ -51,6 +63,30 @@ def log(str)
 
     private
 
+    def with_throttling(job)
+      attempts = 0
+      if job.should_wait?
```

Comment:
Is the `if` only used for logging?

Reply:
Yeah, so far it's only for logging - I think the information the logs provide is quite valuable